README.md

```diff
@@ -160,6 +160,19 @@ Vault instance before starting Vault:
   - `/var/vcap/jobs/vault/tls/other_tls_cert/cert.pem`
   - `/var/vcap/jobs/vault/tls/other_tls_cert/key.pem`
 
+**WARNING** Prior to 1.0.0 release, the `VAULT_SKIP_VERIFY` environment variable is set
+if the vault address contains `https`, so connecting to
+the vault server on 127.0.0.1 (during unseal) would not throw an SSL exception. Since 1.0.0
+release, the environmental variable is no longer  set  by default. There are several possible
+ways to address the situation.
+- If you have **only one** vault node, you can use `properties.vault.addr` to set
```

Comment:
I think we could play nice with multiple nodes on `vault.addr` by doing [the same parsing we're doing on the config](https://github.com/cloudfoundry-community/vault-boshrelease/blob/master/jobs/vault/templates/config/vault.conf.erb#L4-L5) and supporting interpolated variables.